latex2edx/main.py

```diff
@@ -882,7 +882,7 @@ def __str__(self):
                                      eqncontent, re.S) is not None:
                             eqnlabel = re.findall(r'\\label\{(.*?)\}',
                                                   eqncontent, re.S)
-                            eqncontent = re.sub(r'\\label{.*}', r'',
+                            eqncontent = re.sub(r'\\label\{.*?\}', r'',
```

Comment:
Does this type of pattern  work? 

``\{[^\}]*\}``

that is to say, a left brace followed by any number of not-right-braces, followed by a right brace?

I try to avoid getting in to the greedy/lazy debate.